WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -466,8 +464,8 @@ - (void)configureBlogCell:(WPBlogTableViewCell *)cell atIndexPath:(NSIndexPath *
     }
     cell.accessoryType = UITableViewCellAccessoryDisclosureIndicator;
     cell.selectionStyle = self.tableView.isEditing ? UITableViewCellSelectionStyleNone : UITableViewCellSelectionStyleBlue;
-    cell.imageView.layer.borderColor = [UIColor whiteColor].CGColor;
-    cell.imageView.layer.borderWidth = 1.5;
+    cell.imageView.layer.borderColor = self.tableView.separatorColor.CGColor;
```

Comment:
I know we discussed this, but FWIW the border color on the web is white for these icons.